src/stores/grid_renderer_store.ts

```diff
@@ -146,6 +149,10 @@ export class GridRenderer {
         const width = box.width * (percentage / 100);
         ctx.fillRect(box.x, box.y, width, box.height);
       }
+      if (box.overlayStyle?.fillColor) {
```

Comment:
I think this part deserves a test because it's really not clear what It can achieve. we rely on the default behaviour of https://developer.mozilla.org/fr/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation

I wonder if we should "just" allow a darkening factor instead so the intention would be clearer? But I'm not sure either